go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/GoogleContainerTools/kpt
 
 go 1.17
 
+replace github.com/GoogleContainerTools/kpt/porch/api => ./porch/api
```

Comment:
Should this be part of the PR? I think we prefer not having these checked in and instead rely on [golang workspaces](https://go.dev/doc/tutorial/workspaces) for managing the different modules.

Reply:
I'll see if I can separate the CLI changes from the porch changes. I vaguely recall having an issue with doing that, but I need to refresh my memory

Reply:
Or were you suggesting we start using go workspaces now?

Reply:
I split out the CLI changes into a separate PR https://github.com/GoogleContainerTools/kpt/pull/3346, but I had to temporarily remove a test to do so. 

See https://github.com/GoogleContainerTools/kpt/pull/3338#discussion_r915371488